skl2onnx/operator_converters/gaussian_process.py

```diff
@@ -90,9 +90,18 @@ def convert_gaussian_process_regressor(scope, operator, container):
         mean_y = op._y_train_mean.astype(dtype)
         if len(mean_y.shape) == 1:
             mean_y = mean_y.reshape(mean_y.shape + (1,))
-        y_mean = OnnxAdd(y_mean_b, mean_y,
-                         output_names=out[:1],
+        y_mean = OnnxAdd(y_mean_b, mean_y, output_names=out[:1],
                          op_version=opv)
+        # if not hasattr(op, '_y_train_std') or op._y_train_std == 1:
```

Comment:
Commented out code here, can you remove it?

Reply:
Done.